src/sass/components/_product-box.scss

```diff
@@ -29,7 +29,20 @@
     .buttons {
       display: flex;
       justify-content: space-between;
+      visibility: hidden;
     }
+
+
+    }
+    &:hover {
+      
+      .buttons {
+      visibility: visible;
```

Comment:
wcięcie

Reply:
Sformatowałem używając Beautify w VSC. To chyba rozwiązało problem.